### DISCUSSION TITLE
Avoid wasteful duplicates

### DISCUSSION THREAD

**Discussion by xyz342** [facebook/docusaurus#9000] [Q&A] [answered]
Hi!

Is there any way at all to avoid having all the images duplicated for each locale build? For example, each locale folder has an images folder. Located in `build/{LANG}/assets/images` while it could just use the `build/assets/images` without any issue. 

I tried different ways to organise the pictures, but in the end, there will always be an images folder with all the images in it. I'd like the images folder to only be in the `build` folder, and the locale subfolders can just import the pictures from there. 

I have many languages and about 2GB of images, which means an additional 2GB for each language. This is a big issue as Vercel build now fails due to a size limit.

**Comment by Josh-Cena** [comment, marked as answer]
This is a hard issue to fix, and not sure if it can even be called an issue. Currently we emit a fully functional bundle for each locale, because each locale is an independent SPA. You can deploy each locale to a separate domain without sharing any assets. Maybe in the medium/long term we can offer a mode where assets are shared as much as possible, but for now, if you don't plan to localize any of your assets, the best option for you is to make them all static assets and prevent any bundling. For example, if you are using Markdown images, you can put the images under `static/images/foo.png` and then use `![alt message](pathname:///images/foo.png)`. The `pathname://` protocol "tricks" Docusaurus into believing it's an external URL and doesn't bundle anything. This is documented here: https://docusaurus.io/docs/markdown-features/assets#static-assets
